docs/global-config.en-US.md

```diff
@@ -35,7 +35,85 @@ export class AppModule {}
 
 These global configuration would be injected into a service named `NzConfigService` and gets stored.
 
-## Dynamically Chaing Configs
+### Templates
```

Comment:
This title could be more explicit. Something like "Provide Templates in Global Configutation".

Reply:
@wendzhue I've choosen "Provide Template instances", as we already are in global configuration scope and it's sub-sub-sub header. Is it ok?